### ISSUE TITLE
getting started (Ubuntu)

### ISSUE THREAD

**Issue by mloffler** [homeRangeOD/homeRangeOD#1]
I have R version 3.2.3 on my system, but when I try to install "rgdal" from CRAN it tells me the package is not available for my version of R.

Can you please provide a list of a packages / installation steps necessary to run a minimal example opening your data files?

**Comment by Robinlovelace** [comment]
Yes - please follow the Linux install instructions here: https://github.com/r-spatial/sf
